src/test/java/core/basesyntax/dao/StorageDaoImplTest.java

```diff
@@ -0,0 +1,111 @@
+package core.basesyntax.dao;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.FruitTransactionException;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitTransactionValidation;
+import core.basesyntax.service.impl.FruitTransactionValidationImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class StorageDaoImplTest {
+    public static final String DEFAULT_FRUIT = "banana";
+    public static final int DEFAULT_QUANTITY = 100;
+    private FruitTransaction fruitTransaction;
+    private final FruitTransactionValidation transactionValidation
+            = new FruitTransactionValidationImpl();
+    private final StorageDao storageDao = new StorageDaoImpl(transactionValidation);
+
+    @AfterEach
+    void tearDown() {
+        Storage.storage.clear();
+    }
+
+    @BeforeEach
+    void setUp() {
+        fruitTransaction = new FruitTransaction();
```

Comment:
I don't think we need to create a new object before each test, use BeforeAll instead